compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -29,14 +29,11 @@ namespace luci
 namespace
 {
 
-void compute_asym_scale_zp(float min, float max, float *scaling_factor, int64_t *zp)
+void compute_asym_scale_zp(float min, float max, float &scaling_factor, int64_t &zp,
+                           float &nudged_min, float &nudged_max)
```

Comment:
as this function changed from using pointer to reference, how about changing below `asym_wquant_with_minmax` too? @jinevening is it OK?

Reply:
> @jinevening is it OK?

Sure.

Reply:
I will apply it :-)